curvefs/src/metaserver/metastore_fstream.cpp

```diff
@@ -83,11 +83,14 @@ bool MetaStoreFStream::LoadPartition(uint32_t partitionId,
 
     partitionId = partitionInfo.partitionid();
     auto partition = std::make_shared<Partition>(partitionInfo, kvStorage_);
-    partitionMap_->emplace(partitionId, partition);
-    if (!partition->Clear()) {  // it will clear all inodes and dentrys
-        LOG(ERROR) << "Clear partition failed, partitionId = " << partitionId;
-        return false;
-    }
+
+    // FIXME(wuhanqing):
```

Comment:
I think it can delete directly if we ignore handle memory storage.

Reply:
I have move those clear things into `MetaStore::Clear`